tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -204,59 +89,24 @@ module.exports = {
       return this.api.page.FilesPageElement.appSideBar()
     },
     /**
+     * @param {string} resource name
+     * @param {string} resource type (file|folder)
      *
-     * @param {string} fileName
-     */
-    openPublicLinkDialog: async function (fileName) {
-      await this.waitForFileVisible(fileName)
-
-      await this
-        .useXpath()
-        .performFileAction(fileName, FileAction.share)
-        .waitForElementVisible('@linkToPublicLinksTag')
-        .click('@linkToPublicLinksTag')
-        .waitForElementVisible('@publicLinkSideBar')
-        .useCss()
-
-      return this.api.page.FilesPageElement.publicLinksDialog()
-    },
-    /**
-     *
-     * @param {string} fromName
-     * @param {string} toName
-     * @param {boolean} expectToSucceed
+     * @return
      */
-    renameFile: async function (fromName, toName, expectToSucceed = true) {
-      await this.waitForFileVisible(fromName)
-
-      await this.initAjaxCounters()
+    openFileActionsMenu: function (resource, elementType = 'file') {
+      const api = this.api.page.FilesPageElement
```

Comment:
I think doing this is not required in this case. api is used only once so why dont you simply use this.api.FilesPageElement.fileActionMenu()..... in the next line itself.

Reply:
OK, done!